### DISCUSSION TITLE
when doris tablename and column name lowercase， Flink Doris Connector cdc oracle missing data

### DISCUSSION THREAD

**Discussion by copkg** [apache/doris#30296] [A - General / Q&A]
when doris tablename and column name lowercase，use Flink Doris Connector oracle-sync-database，but No data synchronized
if doris tablename and column name uppercase，it's ok.
How to do it to support lowercase table and column names in Doris database, oracle database is uppercase，it can work？

**Comment by JNSimba** [comment]
By default, the whole database synchronization uses json format to import. JSON is strictly case-sensitive for keys, so the column names of doris are lowercase, and the column names of oracle are uppercase and cannot be imported.
